src/components/DpTextArea/DpTextArea.stories.mdx

```diff
@@ -0,0 +1,147 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import DpTextArea from './DpTextArea'
+
+<Meta
+  title="Components/TextArea"
+  component={DpTextArea} />
+
+# DpTextArea
+
+The `DpTextArea` component is used to defines a multi-line text input control.
+
+## Examples
+
+export const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { DpTextArea },
+    template: `<dp-text-area v-bind="$props" />`
+});
+
+### Disabled
+
+<Canvas>
+    <Story
+        name="Disabled"
+        args={{
+            id: 'textAreaId',
+            label: 'Insert the Text.',
+            disabled: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### Hint
+
+<Canvas>
+    <Story
+        name="Hint"
+        args={{
+            id: 'textAreaId',
+            label: 'Insert the Text.',
+            hint: 'Insert the hint.'
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### growToParent
+
+If enabled, classes are applied to let element grow to 100% height of its parent element.
+
+<Canvas>
+    <Story
+        name="growToParent"
+        args={{
+            id: 'textAreaId',
+            label: 'Insert the Text.',
+            growToParent: true
+        }}>
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+### dataDpValidateIf
+
+Use to conditionally validate a required textarea field.
```

Comment:
I think we should explain here what kind of values can be passed to `dataDpValidateIf` so the usage becomes clear.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/380/commits/ff5e2b7c917e1088c8e3ae991ca71cca46e10afc